Makefile.arm64

```diff
@@ -119,11 +119,11 @@ endif
 endif
 
 # Use a72 tunings because Neoverse-N2 is only available
-# in GCC>=9.4
+# in GCC>=10.4
```

Comment:
This should have been 10.1?  It's only a comment, but it's confusing when you read the code below

Reply:
the code says 10.4 too - either a minor version of 4 or major 11 and above